### ISSUE TITLE
Unmet peer dependencies with React 18

### ISSUE THREAD

**Issue by nixjs** [react-boilerplate/redux-injectors#34]
I have a issues when use redux-injectors with React ^18.

<img width="447" alt="image" src="https://user-images.githubusercontent.com/23132483/171783065-1455d11f-33f7-41f2-9b0d-622904603dcf.png">

**Comment by abesto** [comment]
FWIW I've just tested `2.0.0` with React 18 and `react-redux` 8.0.1, and things seem to be working fine AFAICT. _Possibly_ React 18 compatibility is a no-op (and just needs the peer dependencies to be bumped), but I don't think I use all the advanced features of `redux-injectors`, so grain of salt.

**Comment by avx99** [comment]
@nixjs did u solve this issue?

**Comment by nixjs** [comment]
> @nixjs did u solve this issue?

i still have the above problem

**Comment by nixjs** [comment]
> @nixjs did u solve this issue?

I think update [package.json](https://github.com/react-boilerplate/redux-injectors/blob/master/package.json) in redux-injectors:

``` 
"peerDependencies": {
    "react": "^16.6.0 || ^17.0.0 || ^18.0.0",
    "react-dom": "^16.6.0 || ^17.0.0 || ^18.0.0"
  }
  ```
  
   instead of:
   
```
"peerDependencies": {
    "react": "^16.6.0 || ^17.0.0",
    "react-dom": "^16.6.0 || ^17.0.0"
  }
 ```
  
  The problem will be solved.

**Comment by nixjs** [comment]
When do you upgrade new version to support react v18?

**Comment by nixjs** [comment]
> When do you upgrade new version to support react v18?

I forked and supported react v18: https://www.npmjs.com/package/@nixjs23n6/redux-injectors

**Comment by BenLorantfy** [comment]
This is fixed in 2.1.0
